bounties_api/analytics/management/commands/timeline_generator.py

```diff
@@ -165,24 +162,26 @@ def get_noise_bounties(bounties):
     return noise
 
 
-def generate_timeline(time_frame, schema):
+def generate_timeline(time_frame, platform):
     date = time_frame[1]
-    bounty_state_schema = BountyState.objects
-    fulfillment_schema = Fulfillment.objects
-
-    if schema == 'standardSchema':
-        bounty_state_schema = bounty_state_schema.filter(DEFAULT_SCHEMA_QUERY)
-        fulfillment_schema = fulfillment_schema.filter(DEFAULT_SCHEMA_QUERY)
-    elif schema and schema != ALL_SCHEMA:
-        bounty_state_schema = bounty_state_schema.filter(bounty__schemaName=schema)
-        fulfillment_schema = fulfillment_schema.filter(bounty__schemaName=schema)
-
-    bounties_state_frame_day = bounty_state_schema.filter(change_date__range=time_frame, bountyStage=DRAFT_STAGE)
-    bounties_state_frame = bounty_state_schema.filter(change_date__lte=time_frame[1])
-    fulfillment_accepted_frame = fulfillment_schema.filter(accepted_date__lte=time_frame[1])
-    fulfillment_accepted_frame_day = fulfillment_schema.filter(accepted_date__range=time_frame)
-    fulfillment_submitted_frame = fulfillment_schema.filter(fulfillment_created__lte=time_frame[1])
-    fulfillment_submitted_frame_day = fulfillment_schema.filter(fulfillment_created__range=time_frame)
+    bounty_state_platform = BountyState.objects
+    fulfillment_platform = Fulfillment.objects
+
+    if platform == DEFAULT_PLATFORM: 
+        bounty_state_platform = bounty_state_platform.filter(bounty__platform=platform)
+        fulfillment_platform = fulfillment_platform.filter(platform=DEFAULT_PLATFORM)
```

Comment:
Also, it should include the logic | None still - the idea is that if no platform is included it should default to the bounties-network platform. Make sense?